tests/Features/signup.feature

```diff
@@ -0,0 +1,13 @@
+Feature: A Speaker signs up
+
+    Scenario: Speaker successfully signs up
+        Given I am on the signup page
+        When I fill in my account information
+          And I agree to the code of conduct
+          And I press "Signup"
+        Then I should see "Welcome to OpenCFP!"
```

Comment:
I don't think we should test for seeing `OpenCFP` but rather something else that we can use to identify the page.

Since users updating OpenCFP for their own conference will get a failing test suite

Reply:
See my comments above about testing for presence of "OpenCFP".

As an alternative, the step could be `@Then I am greeted as a first-time speaker` that has a step definition that checks presence of message (using config from Kernel) and also checks that the speaker is prompted to fill out the rest of their profile. I think I like this better. Thoughts?

Reply:
`@Then I am greeted as a first-time speaker` looks like a good idea to me.